app/services/Database.scala

```diff
@@ -199,7 +204,11 @@ object DynamoToScala {
         lastModify = Option(item.getString("lastModify")),
         lastModifyBy = Option(item.getString("lastModifyBy")),
         lastModifyByName = Option(item.getString("lastModifyByName")),
-        createdBy = Option(item.getString("createdBy"))
+        createdBy = Option(item.getString("createdBy")),
+        deleted = if (!item.hasAttribute("deleted"))
```

Comment:
I think we need to handle `deleted` inside `toItem` as well, it's the method just above this